lib/home/profile/account/account_settings.dart

```diff
@@ -34,19 +14,10 @@ class AccountSettings extends StatefulWidget {
 }
 
 class _AccountSettingsState extends State<AccountSettings> {
-  late TextEditingController _deleteController;
-
   @override
   void initState() {
-    AccountField.reset();
-    _deleteController = TextEditingController();
     super.initState();
-  }
-
-  @override
-  void dispose() {
-    _deleteController.dispose(); // Dispose of the controller when done
-    super.dispose();
+    AccountField.reset();
   }
```

Comment:
Since `CloseAccount` is now a separate widget, the text controller is no longer needed.